lib/views/go_to_page.dart

```diff
@@ -0,0 +1,174 @@
+    return Container(
+      child: Material(
+        child: Container(
+          margin: new EdgeInsets.only(left: 15.0, top: MediaQuery.of(context).padding.top + 5.0, right: 15.0),
+          //color: Color.fromRGBO(147, 0, 44, 1),
+          padding: EdgeInsets.only(
+            top: 10,
+            right: 35,
+            left: 20,
+          ),
+          child: Column(
+            children: <Widget>[
+              Row(
+                children: <Widget>[
+                  Expanded(
+                    flex: 1, // This flex is for the icon
+                    child: GestureDetector(
+                      onTap: () {
+                        Navigator.pop(context);
+                      },
+                      child: Icon(
+                        Icons.arrow_back,
+                        color: Color.fromRGBO(147, 0, 47, 1),
+                        size: 25,
+                      ),
+                    ),
+                  ),
+                  Expanded(
+                    flex: 10,
+                    child: TypeAheadField(
+                      textFieldConfiguration: TextFieldConfiguration(
+                        style: TextStyle(
+                          fontSize: 18,
+                          fontWeight: FontWeight.w400,
+                          color: Colors.black,
+                        ),
+                        decoration: InputDecoration(
+                          contentPadding:
+                              EdgeInsets.fromLTRB(15.0, 12.0, 20.0, 15.0),
+                          hintText: "origin...",
+                          border: OutlineInputBorder(
+                            borderRadius: BorderRadius.circular(10.0),
+                          ),
+                        ),
+                        controller: this._originTypeAheadController,
+                      ),
+                      suggestionsCallback: (pattern) async {
+                        var roomTitles = new List<String>();
+                        //TODO: add room list here
+                        for (var room in []) {
+                          if (room
+                              .getTitle()
+                              .toUpperCase()
+                              .startsWith(pattern.toString().toUpperCase())) {
+                            roomTitles.add(room.getTitle());
+                          }
+                        }
+                        return roomTitles; //await BackendService.getSuggestions(pattern);
+                      },
+                      itemBuilder: (context, suggestion) {
+                        return Card(
+                          elevation: 2,
+                          child: ListTile(
+                            leading: Icon(Icons.place,
+                                color: Color.fromRGBO(147, 0, 47, 1)),
+                            title: Text(suggestion),
+
+                            //subtitle: Text('\$${suggestion['price']}'),
```

Comment:
Can this comment be deleted? Or is it there for a future reminder?

Reply:
It was deleted.